docs/Customization.md

```diff
@@ -0,0 +1,22 @@
+# Customization
+
+## GraphQL Server
+
```

Comment:
descriptions are fine, best if we can have a list of files to be edited on top, with detailed descriptions below if needed. currently a bit hard to read at a glance.

Reply:
Agree, put a list at the beginning will be good.